### DISCUSSION TITLE
How do I get the keyboard screen?

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [schismtracker/schismtracker#579] [Q&A] [answered]
How do I get this "keyboard" screen working in the player? Kinda nice oversight for things.

![Screenshot_2024-12-24_03-23-52](https://github.com/user-attachments/assets/a9c17810-9176-422c-9283-9f139df8fd42)

**Comment by mrpapersonic** [comment]
Press F5 with a module loaded.

Also, use F1 to get a good idea of what all of the keybinds are and how to use them. Most things are only accessible via the keyboard.

**Reply by JesusGod-Pope666-Info** [reply]
oh come on, you need a module for that? :S

**Comment by JesusGod-Pope666-Info** [comment]
What is the name of this so called module and where can I find it - and.... Like I am getting way to old to this nonsense.

**Comment by JesusGod-Pope666-Info** [comment]
Is the Module inside the standard app or do it need to be installed from somewhere?

**Comment by mrpapersonic** [comment]
A module is a song, you can download them from [Modarchive](https://www.modarchive.org) or [Modland](https://www.modland.com). You should be able to simply drag and drop the song on the window and it will load.

**Reply by JesusGod-Pope666-Info** [reply]
Oh no meant the KEYBOARD SHOWING THING, I have things playing in the player. I saw the tutorial on Youtube. But it is the KEYBOARD thing I do not know how to activate.

**Reply by JesusGod-Pope666-Info** [reply]
Here it is shown at the bottom.
![Screenshot_2024-12-24_03-58-24](https://github.com/user-attachments/assets/9a7d06d3-e6b1-42ae-a906-5e311d82e897)

**Reply by JesusGod-Pope666-Info** [reply]
Buttom, Buttom and Buttom. How do I activate it?
![Screenshot_2024-12-24_04-05-04](https://github.com/user-attachments/assets/f9c0d8c4-6ddd-400e-8b60-a8755e7f062d)
![Screenshot_2024-12-24_04-04-55](https://github.com/user-attachments/assets/5418d18c-4ec6-468a-9b72-683e2313a414)
![Screenshot_2024-12-24_04-04-47](https://github.com/user-attachments/assets/f1ba2d02-a2e7-4f09-aab8-93ebc0e5026e)

**Comment by mrpapersonic** [comment, marked as answer]
press Tab to switch between the menus, and PageUp/PageDown to activate it (btw, those are called note dots ;))

**Reply by JesusGod-Pope666-Info** [reply]
Thanks I will try that, and sorry for being a grumpy old man, pain issues and all that - Thanks yet again.
Glass in your feet for +2 years does not make one happy for sure..... Anyway.

**Reply by JesusGod-Pope666-Info** [reply]
uhhh, very interesting! :) And Note dots, roger!

**Reply by JesusGod-Pope666-Info** [reply]
Is there somehow ways to make them smaller and bigger? It seems the pictures I have they not only change things, but also the size of the 3 areas used.

**Reply by mrpapersonic** [reply]
yep, you can change the sizes of the boxes by holding Alt and pressing the up or down keys.

**Reply by JesusGod-Pope666-Info** [reply]
Do you know Boo's lullaby?
[Boo's Lullaby2.xm.zip](https://github.com/user-attachments/files/18235482/Boo.s.Lullaby2.xm.zip)

**Reply by mrpapersonic** [reply]
No, but I've heard of the artist, he makes some nice tunes :)

**Reply by JesusGod-Pope666-Info** [reply]
Well it is in the Zip, trying to see if I can kinda, use whatever he is using to maybe make practice on some newbie easy tunes.
Just trying to get some more.... experience with these trackers things. Which is why I found the Schism tracker as it had this interesting thing with the note dots and seemed some other things.
I have a Bassoon Tracker running on my website as well...... Picking up some tunes, and getting more accustomed to things.

**Reply by JesusGod-Pope666-Info** [reply]
Anything with those note dots colors? And seems some shapes as well...... hmmmmm.

**Reply by JesusGod-Pope666-Info** [reply]
![Screenshot_2024-12-24_05-00-10](https://github.com/user-attachments/assets/29913c29-4e84-4237-9c39-35b69a45fe09)

**Reply by JesusGod-Pope666-Info** [reply]
> yep, you can change the sizes of the boxes by holding Alt and pressing the up or down keys.

Ahh there we go, had forgotten.

**Comment by JesusGod-Pope666-Info** [comment]
![Screenshot_2024-12-24_05-12-03](https://github.com/user-attachments/assets/6d1b6378-1301-4f10-b046-10bf7c9f2bb8)
![Screenshot_2024-12-24_05-11-57](https://github.com/user-attachments/assets/5a01e453-9f03-4840-aeed-315fa5a11403)

**Comment by JesusGod-Pope666-Info** [comment]
For self: https://manpages.ubuntu.com/manpages/xenial/man1/schismtracker.1.html
